test/unit/RoundsManager.js

```diff
@@ -254,19 +253,22 @@ contract("RoundsManager", accounts => {
     describe("blockHash", () => {
         it("should fail if block is in the future", async () => {
             const latestBlock = await web3.eth.getBlockNumber()
-            await expectThrow(roundsManager.blockHash(latestBlock + 1))
+            // Note that current block = latestBlock + 1, so latestBlock + 2 is in the future
```

Comment:
Actually, the current block number as returned by `web3.eth.getBlockNumber()` should be equal to `latestBlock` here since `eth_call` by default executes in the context of the latest block and not in the context of the pending block to be mined. So, in this case `latestBlock + 1` would be a block number in the future.